manifests/base/configmap-m.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: m-cm
```

Comment:
Can you add the word moneypenny to the name? It helps when looking at all the resources in the environment